src/components/LoadingPage/loadingPage.module.css

```
@@ -1,15 +1,77 @@
-.loading_page {
+.loading_container {
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
+  width: 100%;
+}
+
+.logo_box {
+  width: 180px; 
+  height: 60px;
+  padding: 2rem 0 2.5rem 0;
+  margin: 1rem 0;
+}
+
+.logo_shadow {
+  width: inherit; 
+  height: inherit;
+  border-radius: 0.625rem;
+  background: linear-gradient(90deg, transparent, #EEEEEE, transparent);
+  background-size: 200% 100%;
+  background-position: -100% 0;
+  animation: wave 1.2s infinite;
+}
+
+/* Hide the logo box on screens smaller than 336px */
+@media (max-width: 399px) {
+  .logo_shadow {
+     display: none;
+  }
+  .logo_box {
+     display: none;
+  }
+}
+
+.loading_indicators {
+  display: flex;
   flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+
+/* Styles for larger screens */
+@media (min-width: 768px) {
+  .loading_indicators {
+     max-width: 570px;
+     padding: 2rem 0;
+  }
+}
+ 
+ /* Styles for smaller screens (mobile) */
+ @media (max-width: 767px) {
+  .loading_indicators {
+     min-width: 336px;
+     padding: 2rem 1rem;
+  }
+ }
```

I think you could just do this:

```suggestion
.loading_indicators {
  max-width: var(--awell-activity-max-width);
  padding: var(--awell-activity-padding);
}
```

---

this doesn't work - it's shifts the screens in unexpected ways - I rather update rem to variables but keep it as @media